CyanEmu/Scripts/CyanEmuPlayerController.cs

```diff
@@ -90,6 +92,7 @@ private enum Stance {
         private CollisionFlags collisionFlags_;
         private bool peviouslyGrounded_;
         private bool legacyLocomotion_;
+        private bool updatePosition_;
```

Comment:
Now that this is in a global scope, I would have renamed this to give it more context, but I'm not going to go super nit here and force you to change things just for this.